sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 #### Bugs Fixed
 
 #### Other Changes
+* Client ID will now be included in headers for all requests. See [PR 42104](https://github.com/Azure/azure-sdk-for-python/pull/42104).
```

Comment:
maybe to something like "changed to include client id xxx" to align with others?